python/pyspark/pandas/tests/test_dataframe_slow.py

```diff
@@ -1949,41 +1926,45 @@ def test_between_time(self):
         idx = pd.date_range("2018-04-09", periods=4, freq="1D20min")
         pdf = pd.DataFrame({"A": [1, 2, 3, 4]}, index=idx)
         psdf = ps.from_pandas(pdf)
-        self.assert_eq(
-            pdf.between_time("0:15", "0:45").sort_index(),
-            psdf.between_time("0:15", "0:45").sort_index(),
-        )
+        # This will be addressed from ongoing PR: https://github.com/apache/spark/pull/40370.
```

Comment:
cc @dzhigimont FYI.
I believe your ongoing PRs will be available to merge after this PR is done.